projects/plugins/boost/app/assets/src/js/features/critical-css/status/status.tsx

```diff
@@ -48,37 +48,29 @@ const Status: React.FC< StatusTypes > = ( {
 		);
 	}
 
-	// If my parent has provided override text, show it.
-	if ( overrideText ) {
-		return (
-			<div className={ styles.status }>
-				<div className={ styles.summary }>{ overrideText }</div>
-			</div>
-		);
-	}
-
-	// Otherwise, show the status.
 	return (
-		<div className={ classNames( 'jb-critical-css__meta', styles.status ) }>
+		<div className={ styles.status }>
```

Comment:
Good move removing the `jb-critical-css__meta` class. It's not used for CSS. But it was being used for e2e tests, hence they are breaking. For other parts of the UI during react migration, I have used something like `data-testid="critical-css-meta"` as an attribute. And, then replaced the class selectors in the tests:

`locator('.jb-critical-css__meta')` -> `locator('[data-testid="critical-css-meta"]')`.

Reply:
Yikes! Totally forgot about the tests - thanks for catching this!